overlord/snapstate/snapmgr.go

```diff
@@ -496,6 +496,7 @@ func Manager(st *state.State, runner *state.TaskRunner) (*SnapManager, error) {
 	runner.AddHandler("unlink-current-snap", m.doUnlinkCurrentSnap, m.undoUnlinkCurrentSnap)
 	runner.AddHandler("copy-snap-data", m.doCopySnapData, m.undoCopySnapData)
 	runner.AddCleanup("copy-snap-data", m.cleanupCopySnapData)
+	runner.AddHandler("create-snap-save", m.doCreateSnapSave, m.undoCreateSnapSave)
```

Comment:
I don't think we want a new task just for this, it should be integrated into copy-snap-data other directory creation responsibilities

Reply:
Fixed, I have removed the path, thanks for the review!